frontend/pendingTranslations.ftl

```diff
@@ -23,6 +23,52 @@ fx-desktop-2 = { -brand-name-firefox } for Desktop
 fx-mobile-2 = { -brand-name-firefox } for Mobile
 fx-containers = { -brand-name-firefox } Containers
 
+## Phone Masking FAQ 
+
+phone-masking-faq-question-1 = What is a phone number mask?
```

Comment:
Do these identifiers have to be super meaningful or can they be programmatic like this so I can iterate through? @maxxcrawford would this be a question for translation team or is this something you have already faced?

Reply:
We like them to be somewhat more meaningful. See how we did [other FAQ questions](https://github.com/mozilla-l10n/fx-private-relay-l10n/blob/main/en/app.ftl#L540-L547): 

`faq-question-availability-question = Where is { -brand-name-relay } available?`

This breaks down to `faq-question-[subject/snippet]-[location]` roughly. See the full [`app.ftl`](https://github.com/mozilla-l10n/fx-private-relay-l10n/blob/main/en/app.ftl#L540-L547) file for additional usage examples (double paragraph, variables, etc) 

I know you loop through this number-wise in the FAQ markup, however - if we were to update a question in the future, we'd need a new idea, and this entire naming scheme would break. 

Additionally, for readability, I would put each pair together in the FTL.